frontend/src/modules/app/ToolBar.tsx

```diff
@@ -132,8 +155,13 @@ const alarmConfiguration = (ventilationMode: VentilationMode | null): Array<Alar
 /**
  * ToolBar
  *
- * A container for displaying buttons that handle system changes based on
+ * @component A container for displaying buttons that handle system changes based on
  * various states and conditions such as ventilator state and current page route.
+ *
+ * @param {React.ReactNode} children Adds dynamic items as children for SidebarSlideRoute
```

Comment:
What kind of items should be provided as children? i.e. if we include an item here, what does it do to SidebarSlideRoute?

Reply:
We can provide any valid React Node Element or JSX element as children. 
We are specifically using this in `SidebarSlideRoute.tsx` file to add additional dynamic Views on top of existing Toolbar features. 
In SidebarSlideRoute, this `children` adds Pufferfish Icon & Drawer icon(which toggles between open/closing sidebar) which is shown only for `SidebarSlideRoute` layout.